README.md

```diff
@@ -1,4 +1,8 @@
-![](https://pbs.twimg.com/profile_banners/1103191459409420288/1573207178/1500x500)
```

Comment:
The recent changes for encoding/encoding.go are:

4448f73: “Initial commit”

The file encoding/encoding.go was initially committed with various functions to decode hexadecimal and string representations into integers and byte slices. This includes functions such as DecodeUint64orHex, DecodeUint256orHex, DecodeInt64orHex, and DecodeBytes.![image](https://github.com/user-attachments/assets/c697817a-0fd7-442c-afb6-f52ed00df982)